### PR TITLE
CHAT-520 : missing translations for chat desktop notifications settings

### PR DESCRIPTION
**Pull request by mtrabelsi**

### CODE REVIEW

File: application/src/main/resources/locale/portlet/chat/Resource_en.properties [hunk 1]
```diff
@@ -1,3 +1,18 @@
+exoplatform.chat.desktopNotif.global.notifications=Notifications
+exoplatform.chat.desktopNotif.global.notifyme=Notify me by
+exoplatform.chat.desktopNotif.global.donotdist=Do Not Distrub Notifications
+exoplatform.chat.desktopNotif.global.donotdist.description= Notify me even when my status is Do Not Disturb
```
Comment (thomasdelhomenie) [inline application/src/main/resources/locale/portlet/chat/Resource_en.properties]: Space at the beginning should be removed

File: application/src/main/resources/locale/portlet/chat/Resource_en.properties [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+exoplatform.chat.desktopNotif.global.desktop.description=Display a toast notification on your desktop
+exoplatform.chat.desktopNotif.global.onsite=On-site
+exoplatform.chat.desktopNotif.global.onsite.description=Display a badge over the chat icon in the top bar
+exoplatform.chat.desktopNotif.global.beep=Bips
+exoplatform.chat.desktopNotif.global.beep.description=Play a sound each time I receive a new message
+exoplatform.chat.desktopNotif.local.normal=I want to receive notifications for this room
+exoplatform.chat.desktopNotif.local.silence=I don't want to receive notifications for this room
+exoplatform.chat.desktopNotif.local.alerton=Notify me when someone uses one of these keywords
+exoplatform.chat.desktopNotif.close=Close
+exoplatform.chat.actions.show=Montrer les actions collaboratives
```
Comment (thomasdelhomenie) [inline application/src/main/resources/locale/portlet/chat/Resource_en.properties]: French instead of English

File: application/src/main/resources/locale/portlet/chat/Resource_fr.properties [hunk 3]
```diff
@@ -1,3 +1,17 @@
+exoplatform.chat.desktopNotif.global.notifications=Notifications
+exoplatform.chat.desktopNotif.global.notifyme=Notifier moi par
```
Comment (thomasdelhomenie) [inline application/src/main/resources/locale/portlet/chat/Resource_fr.properties]: "Notifiez-moi par"

File: application/src/main/resources/locale/portlet/chat/Resource_fr.properties [hunk 4]
```diff
@@ -1,3 +1,17 @@
...
+exoplatform.chat.desktopNotif.global.donotdist=ne pas déranger
+exoplatform.chat.desktopNotif.global.donotdist.description=Notifier moi si mon status est 'ne pas déranger'
```
Comment (thomasdelhomenie) [inline application/src/main/resources/locale/portlet/chat/Resource_fr.properties]: Notifiez

File: application/src/main/resources/locale/portlet/chat/Resource_fr.properties [hunk 5]
```diff
@@ -1,3 +1,17 @@
+exoplatform.chat.desktopNotif.global.notifications=Notifications
+exoplatform.chat.desktopNotif.global.notifyme=Notifier moi par
+exoplatform.chat.desktopNotif.global.donotdist=ne pas déranger
```
Comment (thomasdelhomenie) [inline application/src/main/resources/locale/portlet/chat/Resource_fr.properties]: Ne

File: application/src/main/resources/locale/portlet/chat/Resource_fr.properties [hunk 6]
```diff
@@ -1,3 +1,17 @@
...
+exoplatform.chat.desktopNotif.global.donotdist.description=Notifier moi si mon status est 'ne pas déranger'
+exoplatform.chat.desktopNotif.global.desktop=Desktop
+exoplatform.chat.desktopNotif.global.desktop.description=Afficher une popup sur votre bureau
+exoplatform.chat.desktopNotif.global.onsite=On-site
+exoplatform.chat.desktopNotif.global.onsite.description=Afficher un badge
+exoplatform.chat.desktopNotif.global.beep=son
+exoplatform.chat.desktopNotif.global.beep.description=Jouer du son si je reçoi un message
```
Comment (thomasdelhomenie) [inline application/src/main/resources/locale/portlet/chat/Resource_fr.properties]: reçois

File: application/src/main/resources/locale/portlet/chat/Resource_fr.properties [hunk 7]
```diff
@@ -1,3 +1,17 @@
...
+exoplatform.chat.desktopNotif.local.normal=Je reçois les notifications dans cet salon
```
Comment (thomasdelhomenie) [inline application/src/main/resources/locale/portlet/chat/Resource_fr.properties]: ce salon

File: application/src/main/resources/locale/portlet/chat/Resource_fr.properties [hunk 8]
```diff
@@ -1,3 +1,17 @@
...
+exoplatform.chat.desktopNotif.local.silence=Ne pas recevoir des notifications
+exoplatform.chat.desktopNotif.local.alerton=Notifier moi si un mot clé est utilisé
```
Comment (thomasdelhomenie) [inline application/src/main/resources/locale/portlet/chat/Resource_fr.properties]: Notifiez

File: application/src/main/webapp/partials/chat.notification.global.html [hunk 9]
```diff
@@ -1,36 +1,36 @@
 <div class="row">
     <div class="span6">
-        <h4 style="position: relative; left:0%;" class="offset1">Notifications</h4>
+        <h4 class="global offset1" translate="desktopNotif.global.notifications" style="position: relative; left:0%;">Notifications</h4>
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/partials/chat.notification.global.html]: Can't we use the same system already there in the chat instead of adding a new system for translations ? Like in https://github.com/exo-addons/chat-application/blob/d1c2be410811796f8dbc0fc4ecb2b380c25d267f/application/src/main/java/org/exoplatform/chat/portlet/notification/templates/index.gtmpl#L40

File: application/src/main/resources/locale/portlet/chat/Resource_fr.properties (hunk 5 above)
Comment (mtrabelsi) [inline application/src/main/resources/locale/portlet/chat/Resource_fr.properties]: done

File: application/src/main/resources/locale/portlet/chat/Resource_en.properties (hunk 1 above)
Comment (mtrabelsi) [inline application/src/main/resources/locale/portlet/chat/Resource_en.properties]: done

File: application/src/main/resources/locale/portlet/chat/Resource_fr.properties (hunk 4 above)
Comment (mtrabelsi) [inline application/src/main/resources/locale/portlet/chat/Resource_fr.properties]: done

File: application/src/main/resources/locale/portlet/chat/Resource_fr.properties (hunk 6 above)
Comment (mtrabelsi) [inline application/src/main/resources/locale/portlet/chat/Resource_fr.properties]: done

File: application/src/main/resources/locale/portlet/chat/Resource_fr.properties (hunk 7 above)
Comment (mtrabelsi) [inline application/src/main/resources/locale/portlet/chat/Resource_fr.properties]: done

File: application/src/main/resources/locale/portlet/chat/Resource_fr.properties (hunk 8 above)
Comment (mtrabelsi) [inline application/src/main/resources/locale/portlet/chat/Resource_fr.properties]: done

File: application/src/main/webapp/partials/chat.notification.global.html (hunk 9 above)
Comment (mtrabelsi) [inline application/src/main/webapp/partials/chat.notification.global.html]: No way! We used the the SPA techniques so no JUZU render can have place here, only we manipulate the DOM

Comment (thomasdelhomenie) [inline application/src/main/webapp/partials/chat.notification.global.html]: I don't mean to change the current technique of partial pages, but just to serve the partial pages by Juzu instead in order to be consistent in i18n management. Isn't it possible ?

File: application/src/main/webapp/js/chat.js [hunk 10]
```diff
@@ -354,6 +354,7 @@ var chatApplication = new ChatApplication();
 
 
     var handleGlobalNotifLayout = function () {
+        //desktopNotification.translate(jqchat,'.global');
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/chat.js]: To remove

File: application/src/main/webapp/js/chat.js [hunk 11]
```diff
@@ -426,10 +427,16 @@ var chatApplication = new ChatApplication();
     //global desktop notification settings
     $("#configButton").on("click", function() {
       chatApplication.configMode = true;
-      jqchat("#chats").load("/chat/partials/chat.notification.global.html", handleGlobalNotifLayout);
+    //  jqchat("#chats").load("/chat/partials/chat.notification.global.html", handleGlobalNotifLayout);
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/chat.js]: To remove

File: application/src/main/webapp/js/chat.js [hunk 12]
```diff
@@ -86,7 +86,7 @@ var chatApplication = new ChatApplication();
      */
 
     $("#PlatformAdminToolbarContainer").addClass("no-user-selection");
-
+    $("#global").hide();
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/chat.js]: I think we should use a more meaningful id, such as "globalPreferences".

File: application/src/main/webapp/partials/chat.notification.global.html [hunk 13]
```diff
@@ -1,36 +1,39 @@
-<div class="row">
-    <div class="span6">
-        <h4 style="position: relative; left:0%;" class="offset1">Notifications</h4>
-        <div id="id2" class="offset2 notif-trigger" style="position: relative; left: -6%;">
-            <input notif-trigger="notify-even-not-distrub" type="checkbox" class="switchBtnLabelOn yesno staus-true" />
-            <span class="channel-label"><b>Do Not Distrub Notifications</b></span>
-            <span class="channel-label clearfix" style="position: relative; left: 18%;"><i>Notify me even when my status is Do Not Disturb</i></span>
+<div id="global">
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/partials/chat.notification.global.html]: Is this file still necessary ?

File: application/src/main/webapp/js/chat.js (hunk 12 above)
Comment (mtrabelsi) [inline application/src/main/webapp/js/chat.js]: I'm still working on this, this is not a final commit

File: application/src/main/webapp/partials/chat.notification.global.html (hunk 9 above)
Comment (mtrabelsi) [inline application/src/main/webapp/partials/chat.notification.global.html]: done

File: application/src/main/java/org/exoplatform/chat/portlet/chat/templates/index.gtmpl [hunk 14]
```diff
@@ -1,3 +1,74 @@
...
+<div class="row">
+    <div class="span8 offset1 chat-room-config">
+
+        <input type="radio" room-notif-trigger="normal"  name="optionsRadios" id="optionsRadios1" value="option1" checked />
+        <span class="label-head"> Normal</span> <br/>
+        <span class="label-text local">&{exoplatform.chat.desktopNotif.local.normal}</span><br/>
+
+        <input type="radio" room-notif-trigger="silence"  name="optionsRadios" id="optionsRadios2" value="option2" />
+        <span class="label-head"> Silence </span><br/>
+        <span class="label-text local" translate="desktopNotif.local.silence">&{exoplatform.chat.desktopNotif.local.silence}</span><br/>
```
Comment (thomasdelhomenie) [inline application/src/main/java/org/exoplatform/chat/portlet/chat/templates/index.gtmpl]: I think the translate attribute is not necessary anymore.

File: application/src/main/webapp/js/desktopNotification.js [hunk 15]
```diff
@@ -143,7 +143,19 @@ var desktopNotification = (function() {
     return highlightedMsg;
   }
 
+
+  var translate = function($,component){
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/desktopNotification.js]: Still used ?

**Comment by MrHanachoo** [comment]
replaced by  #78
